charts/scalardl/values.yaml

```diff
@@ -140,6 +140,43 @@ ledger:
     # -- The secret key of a Ledger private key
     ledgerPrivateKeySecretKey: private-key
 
+  # -- The ledger.properties is created based on the values of ledger.scalarLedgerConfiguration by default. If you want to customize ledger.properties, you can override this value with your ledger.properties.
+  # @default -- The minimum template of ledger.properties is set by default.
```

Comment:
```suggestion
  # @default -- The default minimum necessary values of ledger.properties are set. You can overwrite it with your own ledger.properties.
```

Reply:
Thank you for the suggestion!
I applied it.